src/main/app/src/hooks/browseActivitiesFilter.js

```diff
@@ -27,25 +27,41 @@ const filterActivities = (links, activityType, activityTypes, linkFilters, updat
 */
 const filterGames = (game, filters) => {
   var input = parseInt(filters.numOfPlayers,10);
-    var min = parseInt(game.minPlayer,10);
-    var max = parseInt(game.maxPlayer,10);
-    
-    return (min === 0 && max === 0) ||                 // If there are no restrictions on the number of players
-        (min === 0 && max >= input) ||              // If no restriciton on minPlayer, but there is a max limit satisfied
-        (max === 0 && min <= input) ||              // If no restriction on maxPLayer, but minimum limit is satisfied
-        (min <= input && max >= input);         // If both are non-zero, and the game players are within limit. 
-    
-    // Otherwise don't return. 
+  var min = parseInt(game.minPlayer,10);
+  var max = parseInt(game.maxPlayer,10);
+  
+  return (min === 0 && max === 0) ||                 // If there are no restrictions on the number of players
+    (min === 0 && max >= input) ||              // If no restriciton on minPlayer, but there is a max limit satisfied
+    (max === 0 && min <= input) ||              // If no restriction on maxPLayer, but minimum limit is satisfied
+    (min <= input && max >= input);         // If both are non-zero, and the game players are within limit. 
+  
+  // Otherwise don't return. 
 }
 
 /** Filter list of articles. */
 const filterArticles = (article, filters) => {
   return filters.articleType === '*' || article.type === filters.articleType;
 }
 
-/** Filter list of videos. */
+/** 
+  Filter list of videos.
+  Short videos duration  -  [0,900]
+  Medium length videos - [901,1800]
+  Large length videos- (>=1801)
+*/
 const filterVideos = (video, filters) => {
```

Comment:
The filterVideo function doesn't filter by video category with this change. Wouldn't adding a check for what type of filter it is and then filtering either by category or by duration then be better?

Reply:
Yes for sure. I wanted to add that when Trinity adds the categories filter inside this.